gcp/v2/README.md

```diff
@@ -0,0 +1,83 @@
+# Kubeflow on KCC Installation Guide
+
+This instruction explains how to setup kubeflow on top of KCC and ASM.
```

Comment:
It'd be nice to give a bit more context. For example, is the following correct?

```
This instruction explains how to set up Kubeflow on top of Config Connector (KCC) and Anthos Service Mesh (ASM).
Compared with the currently documented GCP deployment, this architecture uses KCC instead of Deployment Manager, and managed Istio in the form of ASM instead of open source Istio.
```